chainio/clients/avsregistry/reader.go

```diff
@@ -278,14 +282,14 @@ func (r *ChainReader) GetOperatorStakeInQuorumsOfOperatorAtCurrentBlock(
 // This function computes the total weight of the operator in the quorum quorumNumber.
 func (r *ChainReader) WeightOfOperatorForQuorum(
 	opts *bind.CallOpts,
-	quorumNumber types.QuorumNum,
+	quorumNumber uint8,
```

Comment:
Why the revert?

Reply:
I realized everything else not written by me was using uint8 not the types, just to maintain consistency